libafl/src/state/mod.rs

```diff
@@ -1081,7 +1081,7 @@ where
             rand,
             executions: 0,
             imported: 0,
-            start_time: Duration::from_millis(0),
+            start_time: libafl_bolts::current_time(),
```

Comment:
Is this even used anywhere @tokatoka

Reply:
it is not used anywhere but it is useful right?
every fuzzer should know when it started fuzzing

Reply:
Last restart could also be interesting fwiw